assets/src/block-templates/issues/template.js

```diff
@@ -66,7 +66,7 @@ const template = ({
     }, [
       ['core/heading', {
         level: 2,
-        placeholder: titlePlaceholder,
+        content: titlePlaceholder,
```

Comment:
This is to be consistent with other templates, also the idea is that this "placeholder" could potentially be used as the final title